### PR TITLE
Check for MCrypt in SysInfo

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/9507.
#### Summary of Changes

Adds php mcrypt extension information to sysinfo.

image
#### Testing Instructions
1. Use Joomla 3.5.0
2. Aply patch
3. Go to sysinfo and check if in php settings separator there is a new line "MCrypt Enabled"
#### Additional information

PR for adding this info to the installer: https://github.com/joomla/joomla-cms/pull/9508

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_admin.ini [hunk 1]
```diff
@@ -112,6 +112,7 @@ COM_ADMIN_LOG_DIRECTORY="(Log folder)"
 COM_ADMIN_MAGIC_QUOTES="Magic Quotes"
 COM_ADMIN_MAX_INPUT_VARS="Maximum Input Variables"
 COM_ADMIN_MBSTRING_ENABLED="Multibyte String (mbstring) Enabled"
+COM_ADMIN_MCRYPT_ENABLED="MCrypt Enabled"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_admin.ini]: This should be Mcrypt not MC

**Comment by mbabker** [comment]
Aside from the language string comment, looks fine to me.

**Comment by brianteeman** [comment]
Thanks for correcting the string

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 5982059a708f1c0754f1e7707ffce3ef41d67a3f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9532">issues.joomla.org/joomla-cms/9532</a>.</sub>

**Comment by brianteeman** [comment]
Taking @mbabker comment as a successful test setting to RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9532">issues.joomla.org/joomla-cms/9532</a>.</sub>

**Pull request merged**